SonarQube.Common/Serializer.cs

```diff
@@ -37,7 +37,7 @@ public static class Serializer
 
             // Serialize to memory first to reduce the opportunity for intermittent
             // locking issues when writing the file
-            using (MemoryStream stream = new MemoryStream())
+            MemoryStream stream = new MemoryStream();
```

Comment:
I agree the rule is a bit dubious: well-written dispose methods handle being called multiple times. I've checked the implementation and the stream writer does flush and close the underlying string (unless you pass leaveOpen=true when you construct it). Still, calling twice won't hurt.